### PR TITLE
PAE-433 - Add email VUE theme 

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
Add vue style to pearson-vue-theme emails

Screenshots
-----------------

image

**Tickets**: [PAE-433](https://pearsonadvance.atlassian.net/browse/PAE-443)

### CODE REVIEW

**Review by ivanvgh — APPROVED** [review]
LGTM

File: edx-platform/pearson-vue-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 1]
```diff
@@ -171,10 +173,10 @@
                     <tr>
                         <!-- COPYRIGHT -->
                         <td>
-                            &copy; {% now "Y" %} {{ platform_name }}, {% trans "All rights reserved" as tmsg %}{{ tmsg | force_escape }}.<br/>
-                            <br/>
-                            {% trans "Our mailing address is" as tmsg %}{{ tmsg | force_escape }}:<br/>
-                            {{ contact_mailing_address }}
+                          Copyright &copy; {% now "Y" %} Pearson Education, Inc. or its affiliate(s). All rights reserved.
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/ace_common/edx_ace/common/base_body.html:176]: Is it necessary to add the Copyright text? Isn't enough with &copy;? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/user_authn/edx_ace/passwordreset/email/body.html [hunk 2]
```diff
@@ -1,95 +1,60 @@
...
-
-
-                 {% endif %}
-
-
-
-
-
-            </div>
+                {# xss-lint: disable=django-trans-missing-escape #}
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/user_authn/edx_ace/passwordreset/email/body.html:36]: Why is this necessary? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/ace_common/edx_ace/common/base_body.html:176]: So it was required

File: edx-platform/pearson-vue-theme/lms/templates/user_authn/edx_ace/passwordreset/email/body.html (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/user_authn/edx_ace/passwordreset/email/body.html:36]: the next line not contains forced_escaped, so it is necessary, https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/preventing_xss/preventing_xss.html#django-trans-missing-escape

**Review by Squirrel18 — APPROVED** [review]
